meteor/client/ui/SegmentContainer/withResolvedSegment.ts

```diff
@@ -100,10 +100,15 @@ export interface IProps {
 	showDurationSourceLayers?: Set<ISourceLayer['_id']>
 }
 
+export interface SegmentNoteCounts {
+	criticial: number
+	warning: number
+}
+
 export interface ITrackedProps {
 	segmentui: SegmentUi | undefined
 	parts: Array<PartUi>
-	segmentNotes: Array<SegmentNote>
+	segmentNotes: SegmentNoteCounts
```

Comment:
I think it would be more explanatory if this interface property was also renamed